src/coreclr/vm/interpexec.cpp

```diff
@@ -1213,6 +1214,32 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                     ip += 5;
                     goto CALL_INTERP_SLOT;
                 }
+                case INTOP_NEWOBJ_VAROBJSIZE:
+                {
+                    returnOffset = ip[1];
+                    callArgsOffset = ip[2];
+                    methodSlot = ip[3];
+
+                    // FIXME: Duplicated code from CALL_INTERP_SLOT
+                    size_t targetMethod = (size_t)pMethod->pDataItems[methodSlot];
+                    MethodDesc *pMD = nullptr;
+                    if (targetMethod & INTERP_METHOD_HANDLE_TAG)
```

Comment:
Should this be assert instead? If this is not set, the `pMD` is null and the `TryGetMultiCallableAddrOfCode` below will crash.